Assets/MixedRealityToolkit.SDK/Inspectors/UX/ManipulationHandler/ManipulationHandlerInspector.cs

```diff
@@ -174,7 +174,22 @@ public override void OnInspectorGUI()
 
         private void DrawDeprecated()
         {
-            EditorGUILayout.HelpBox("This component is deprecated. Please migrate object to up to date version", UnityEditor.MessageType.Warning);
+            System.Collections.Generic.List<System.Type> requiringTypes = new System.Collections.Generic.List<System.Type>();
```

Comment:
```suggestion
            List<System.Type> requiringTypes = new List<System.Type>();
```

Reply:
I think that doing so will also require a using namespace earlier - I very rarely see fully qualified names for basic collections

Reply:
yeah, I thought since it's the only place I'd put it this way, I'll change it